internal/manifest/manifest.go

```diff
@@ -175,13 +135,13 @@ func (m *Manifest) Validate() error {
 	return nil
 }
 
-// SNPValidateOpts returns validate options populated with the manifest's
-// SNP reference values and trusted measurement.
-func (m *Manifest) SNPValidateOpts() (*validate.Options, error) {
+// AKSValidateOpts returns validate options populated with the manifest's
+// AKS reference values and trusted measurement.
+func (m *Manifest) AKSValidateOpts() (*validate.Options, error) {
```

Comment:
If `AKS` is `nil` and `BareMetalTDX` is not nil and valid, this function will still access `AKS` and cause a null-pointer panic.